client/src/components/Input/Input.styles.tsx

```diff
@@ -1,28 +1,90 @@
 import styled from 'styled-components';
 
 import theme from 'styles/theme';
-import { colorChange } from 'styles/mixins';
+import { colorChange, radioWrapperRoot } from 'styles/mixins';
 
 import { StylesProps } from 'types';
 
-export const Input = styled.input<StylesProps>`
+const inputRoot = (props: StylesProps) => `
   width: 100%;
   height: 3rem;
-  padding: 0 2rem 0 4.25rem;
+  padding: 0 2rem;
   border: none;
   border-radius: ${theme.shape.borderRadius};
-  ${(props: StylesProps) =>
-    colorChange(
-      props,
-      'background',
-      theme.palette.componentBackgroundLight,
-      theme.palette.componentBackgroundDark,
-      'all'
-    )}
-  ${(props: StylesProps) =>
-    colorChange(props, 'color', theme.palette.black, theme.palette.white, 'all')}
-
-  &:focus {
-    outline: none;
+  ${colorChange(
+    props,
+    'background',
+    theme.palette.componentBackgroundLight,
+    theme.palette.componentBackgroundDark,
+    'all'
+  )}
+  ${colorChange(props, 'color', theme.palette.black, theme.palette.white, 'all')}
+`;
+
+export const Input = styled.input<StylesProps>`
+  ${props => inputRoot(props)}
+`;
+
+export const Textarea = styled.textarea<StylesProps>`
+  ${props => inputRoot(props)}
+  height: unset;
+  padding: 1rem 2rem;
+  overflow: hidden;
+  resize: none;
+`;
+
+export const InputGroup = styled.div`
+  div:not(:last-of-type) {
+    margin-right: 1rem;
   }
 `;
+
+export const RadioWrapper = styled.div<StylesProps>`
+  ${radioWrapperRoot}
+
+  label {
+    margin-left: 0.5rem;
+  }
+
+  label::before,
+  label::after {
+    content: '';
+    position: absolute;
+    border-radius: 50%;
+    top: 50%;
+  }
+
+  label::before {
+    left: -1.5px;
+    width: 1rem;
+    height: 1rem;
+    background-color: transparent;
+    border: ${({ themeType }) =>
+      themeType === 'light'
+        ? `1.75px solid ${theme.palette.grey.light}`
+        : `1.75px solid ${theme.palette.componentBackgroundDark}`};
```

Comment:
can't you use ternary operator inside `1.75px solid ...` string?

Reply:
I can, thanks.